lib/src/radial_gauge/radial_track.dart

```diff
@@ -121,6 +122,11 @@ class RadialTrack {
   /// The [trackStyle] property is used to customize the track of the Radial Gauge.
   ///
   final TrackStyle trackStyle;
+
+  ///
+  /// The [mapTrackLabel] property is used to customize the track labels of the Radial Gauge.
```

Comment:
Can we add and example code here ?

Reply:
Thanks for your reactivity. I add some example here, in the example directory and a screenshot in the README.